app/views/questions/new.html.erb

```diff
@@ -5,18 +5,33 @@
         <div class="box">
           <h2 class="title-no-margin center">New Question</h2>
           <br>
-            <%= form_for @question do |f| %>
+            <%= form_for(@question) do |f| %>
               <div class="form-group">
-                <%= f.text_field :text, placeholder: 'Question', class: 'form-control', autofocus: true %>
+                <%= f.text_field :text, placeholder: 'Question',  class: 'form-control', autofocus: true %>
               </div>
               <div class="form-group">
-                <%= f.label :qset_id, 'Qset', {class: 'control-label'} %>
-                <%= f.collection_select(:qset_id, Qset.find(current_user.org).descendants, :id, :name, {}, {class: 'form-control'}) %>
+                <button type="button" class="btn btn-primary btn-theme btn-generate-q" data-toggle="modal" data-target="#modal-launch">
+                  Choose Qset
+                </button>
+                <% if !params[:qset].nil? %>
```

Comment:
I notice that after submitting a question, the form resets completely -- it looks like the changes to the view don't respect the cookie stuff in the controller. I think it _should_ remember your most recent one, at least if you've just created a new question, because you may want to create multiple questions in that qset, and having to repeatedly select the same qset isn't very user-friendly. If you decide to move away from using the cookie stuff in the controller, we need to remove it (i.e. if you just go with setting the `:qset` param each time, so that it's picked up by this view code).